### PR TITLE
Run generate-files periodically to reconcile midstream manifests

### PR DESCRIPTION
**Pull request by pierDipi**
Since we're generating files based on a floating branch in midstream,
we need to reconcile those files periodically.

This flow runs periodically or on-demand.

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

**Comment by pierDipi** [comment]
I'll test this workflow once merged.

**Comment by pierDipi** [comment]
This is an example PR https://github.com/openshift-knative/eventing-hyperfoil-benchmark/pull/42 that is generated by a similar config.

File: .github/workflows/validate.yaml [hunk 1]
```diff
@@ -40,6 +43,23 @@ jobs:
...
+      - name: Create Pull Request
+        if: github.event_name != 'pull_request'
+        uses: peter-evans/create-pull-request@v4
+        with:
+          branch: auto/update-generated-files
+          title: "Run make generated-files"
+          commit-message: "Run make generated-files"
+          delete-branch: true
+          body: |
+            Generated by running `make generated-files`
```
Comment (matzew) [inline .github/workflows/validate.yaml:61]: very nice idea to keep this up-to-date!

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**